### ISSUE TITLE
Display postcode on address information panel, if available

### ISSUE THREAD

**Issue by UAb5eSMn** [osmandapp/Osmand#5705]
When tapping on a house on the map, the house number, street name and city/neighbourhood name are displayed:
![address-1](https://user-images.githubusercontent.com/5942222/42907353-0bb35374-8acd-11e8-930c-d8360d26a6c5.png)
Would it be possible to also display the postcode information on that same panel? It's available via search:
![address-2](https://user-images.githubusercontent.com/5942222/42907361-10b1673a-8acd-11e8-806a-680b71549938.png)

**Comment by vshcherb** [comment]
Not today cause the information is structured in the opposite way i.e. we can find address by zip code but we can't find zip code by address or location

**Comment by Peter0** [comment]
Some objects are tagged with a zip field. So I propose to display it, at least if the zip field is available in the properties of the namely object.

**Comment by Peter0** [comment]
In addition it is annoying because under "details" there is a menu item that is about copying the address of the object in the clipboard. And the zip belongs firmly to the address (at least in Germany).

**Comment by Eiernase** [comment]
I agree. This should not be "nice to have". When I look up a place to get the address, I expect the post code to be there as well.
I've been needing to go to the OpenStreetMap website manually to look it up.
It has been eight years since this was opened. An update is certainly due.

**Comment by NastyFlytrap** [comment]
Are you seriously telling me EIGHT YEARS later this is not a feature implemented?! OSM itself has this data, i can literally go to openstreetmap and selecting a house address tells me its postal code, what do you mean you cant do it?!

I swear to god this application is the best we have and it sucks ass. What are you all even working on if basic ass features like these are 'out of reach'?!

**Comment by Peter0** [comment]
> Are you seriously telling me EIGHT YEARS later this is not a feature implemented?! OSM itself has this data, i can literally go to openstreetmap and selecting a house address tells me its postal code, what do you mean you cant do it?!
> 
> I swear to god this application is the best we have and it sucks ass. What are you all even working on if basic ass features like these are 'out of reach'?!

I protest against this rude language, albeit factually correct.

**Comment by NastyFlytrap** [comment]
> > Are you seriously telling me EIGHT YEARS later this is not a feature implemented?! OSM itself has this data, i can literally go to openstreetmap and selecting a house address tells me its postal code, what do you mean you cant do it?!
> > I swear to god this application is the best we have and it sucks ass. What are you all even working on if basic ass features like these are 'out of reach'?!
> 
> I protest against this rude language, albeit factually correct.

I apologise. I just asked for so many features over the last 5 years and got told its unreasonable or impossible, even though i wasnt asking for things like live road updates, or live public transportation data. Just things like 'less clutter', because as it stands, there's just black dots, trees, churches, traffic lights, and so many other things shows on the map it just makes it feel cluttered. Or the ability for the public transport routing to not give you night buses at 2pm all the time! And every time i was its never going to happen. The time i asked for the maps to be faster to load? Yeah, same. Openly told they cant do it. And they have been doing exactly that for the last few years and its not bad now. I dont know if its just a handful of insufferable contributors making me hate the whole lot but OSMAnd is among the single most insufferable volunteer lead projects i have used and every time i have to interact with these people i get the strong urge to just go back to google maps because of how unpleasant it is to deal with these people!

**Comment by hans-fritz-pommes** [comment]
Related to #15541
